ports/arrayfire/portfile.cmake

```
@@ -0,0 +1,73 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO arrayfire/arrayfire
+    REF 646bdeed1557abfdfe6735b9d20377c5b08a12da # v3.7.3
+    SHA512 8af15c25bc46235c0554dad963d94c36ae078e2ad7e65ccbbe101e9f06c33618b52207c0a26b8dd56b3d7c9e0032ea75d08f18a5644eadbf56fefe5952aff699
+    HEAD_REF master
+    PATCHES build.patch
+)
```

```suggestion
vcpkg_from_github(
  OUT_SOURCE_PATH SOURCE_PATH
  REPO arrayfire/arrayfire
  REF 646bdeed1557abfdfe6735b9d20377c5b08a12da # v3.7.3
  SHA512 8af15c25bc46235c0554dad963d94c36ae078e2ad7e65ccbbe101e9f06c33618b52207c0a26b8dd56b3d7c9e0032ea75d08f18a5644eadbf56fefe5952aff699
  HEAD_REF master
  PATCHES build.patch
  )
```
keepin that indentation